Shared/Samples/Run valve isolation trace/RunValveIsolationTraceView.swift

```diff
@@ -0,0 +1,182 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct RunValveIsolationTraceView: View {
+    /// The view model for the sample.
+    @StateObject var model = Model()
+    
+    /// The last locations in the screen and map where a tap occurred.
+    @State var lastSingleTap: (screenPoint: CGPoint, mapPoint: Point)?
+    
+    /// A Boolean value indicating if the configuration sheet is presented.
+    @State var isConfigurationPresented = false
+    
+    var body: some View {
+        MapViewReader { mapViewProxy in
+            MapView(
+                map: model.map,
+                graphicsOverlays: [model.parametersOverlay]
+            )
+            .onSingleTapGesture { screenPoint, mapPoint in
+                lastSingleTap = (screenPoint, mapPoint)
+            }
+            .overlay(alignment: .top) {
+                Text(model.statusText)
+                    .frame(maxWidth: .infinity, alignment: .center)
+                    .padding(10)
```

Comment:
Conceptually, the text should be padded before the frame is altered.

```suggestion
                    .padding(10)
                    .frame(maxWidth: .infinity, alignment: .center)
```